demo/chbench/mzcompose.yml

```diff
@@ -174,6 +174,12 @@ services:
     volumes:
       - chbench-gen:/gen
 
+  kafkasnap:
+    build: kafkasnap
```

Comment:
as mentioned above, I'd prefer for this to be an `mzbuild` so that we don't need to rebuild things on every new init, and we can be sure that folks are using the exact same image.

Reply:
Okay, I can do that.